src/Middleware/ChaosHandler.php

```diff
@@ -45,6 +55,7 @@ class ChaosHandler
      */
     public function __construct(callable $nextHandler, ?ChaosOption $chaosOption = null)
     {
+        $this->tracer = Globals::tracerProvider()->getTracer((new ObservabilityOption())->getTracerInstrumentationName());
```

Comment:
Missing Kiota HTTP package version

Reply:
All handlers should check the request options in `__invoke()` for any ObservabilityOption overrides before initialising the tracer.